src/translations/iw.translation

```diff
@@ -528,7 +528,7 @@ day: &apos;ום&apos;</label>
         <name>labelShiftClosed</name>
     </customLabels>
     <customLabels>
-        <label>מאושר: {0}&amp;nbsp;&amp;nbsp;זמין: {1}</label>
+        <label>מאושר: {0} - זמין: {1}</label>
```

Comment:
It looks like the Hebrew translation got flipped around. Looks like the original and the changed translation do not have the curly braces properly around the numbers either. Sometimes when working with the hebrew in the translation file in your IDE will cause letters to move around. Copy and pasting values vs typing them usually works well.

Reply:
Looks like that is just how it is being displayed in github and in Salesforce but when I actaully view the Hebrew on the calendar the merge fields are working ok.
![screen shot 2019-02-12 at 11 05 21 am](https://user-images.githubusercontent.com/450473/52653748-3afb8b00-2eb6-11e9-885e-c0c24f9da691.png)

Reply:
That's a weird one - in VS Code it's rendered like this:

![screen shot 2019-02-12 at 1 37 54 pm](https://user-images.githubusercontent.com/1134606/52666397-70f73a00-2ecb-11e9-8d3e-01993cbbada9.png)